script/build.js

```diff
@@ -422,7 +422,12 @@ if (options.watch) {
     historyApiFallback: {
       rewrites: [
         { from: '^/track-claims(.*)', to: '/track-claims/' },
-        { from: '^/education/apply-for-education-benefits/application(.*)', to: '/education/apply-for-education-benefits/application/' },
+        { from: '^/education/apply-for-education-benefits/application/1990(\/.*)?$', to: '/education/apply-for-education-benefits/application/1990/' },
```

Comment:
Thanks @cvalarida for making the regex solution here more elegant.

Reply:
The redirect currently adds several trailing `/`'s briefly before navigating to `/introduction`